src/components/table-of-contents/_macro-options.md

```diff
@@ -1,10 +1,13 @@
+| ----------------- | -------------------------------------------------------- | ----------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| title             | string                                                   | true                          | A title for the component, for example, “Contents”                                                                                                                                                                 |
+| ariaLabel         | string                                                   | false                         | Descriptive landmark ARIA label to give a screen reader user greater understanding of its purpose. Defaults to “Table of contents”.                                                                                |
+| skipLink          | `Skip to content` [_(ref)_](/components/skip-to-content) | false                         | Settings for the skip to content link that allows users to avoid reading out the table of contents on each page. Required for any table of contents repeated on multiple pages, such as a [guide](/patterns/guide) |
+| lists             | array`<list>`                                            | true (unless `itemsList` set) | An array of [lists](#lists) for the component                                                                                                                                                                      |
+| itemsList         | array`<itemsList>`                                       | true (unless `lists` set)     | An array of [list items](#itemslist) for the component                                                                                                                                                             |
+| relatedLinksTitle | string                                                   | false                         | Title for the related links                                                                                                                                                                                        |
+| relatedLinksList  | array`<itemsList>`                                       | false                         | An array of [list item links](#itemslist)                                                                                                                                                                          |
+| button            | object                                                   | false                         | An optional object of onsButton options to include a button below the table of contents                                                                                                                            |
```

Comment:
Where you have object here I think we need to change it to similar to the skipLink param and have the button component mentioned